pkg/vm/engine/tae/tables/updates/delchain.go

```diff
@@ -274,6 +274,11 @@ func (chain *DeleteChain) AddMergeNode() txnif.DeleteNode {
 func (chain *DeleteChain) CollectDeletesInRange(
 	startTs, endTs types.TS,
 	rwlocker *sync.RWMutex) (mask *nulls.Bitmap, err error) {
+	var needWaitFound bool
+
+Retry:
```

Comment:
why this need goto?   a loop is not good enough?

Reply:
fixed